### PR TITLE
And rule for disabling wildcard imports

### PR DESCRIPTION
**Pull request by csdodd**

### CODE REVIEW

File: android-studio-style-guide.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Android Studio Style Guide
+
+The intention of this guide is to provide a set of conventions that ensure Android Studio is configured correctly across developers machines.
+
+## Disable wildcard imports
+
+In certain circumstances Android Studio will wildcard import by default. That default behaviour should be disabled as follows.
+
+Go to <kbd>Preferences</kbd> -> <kbd>Editor</kbd> -> <kbd>Code Style</kbd> -> <kbd>Kotlin</kbd>
```
Comment (designatednerd) [inline android-studio-style-guide.md:9]: ooh i was not aware of the `kbd` tag. Neat!

File: android-studio-style-guide.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - open <kbd>Imports</kbd> tab
+    - select `Use single name import` (all of them);
+    - remove `import java.util.*` from `Packages to Use Import with '*'`.
+	- Keep enabled (or add if not there) `import kotlinx.android.synthetic.*` including with subpackages.
```
Comment (designatednerd) [inline android-studio-style-guide.md:13]: Might be worth noting you have to do this for both Java (if you're using it) and Kotlin

Comment (csdodd) [reply android-studio-style-guide.md:13]: What's Java?

**Comment by anskaal** [comment]
`(all of them)` means this one too?
image

**Comment by csdodd** [comment]
~It does, I added the wrong screenshot 🙈~ Ignore me. Getting confused. But yes, use Single name import for that one as well.

File: android-studio-style-guide.md (hunk 2 above)
Comment (csdodd) [reply android-studio-style-guide.md:13]: Actually, it looks like Java currently defaults to something sane-ish. It only wildcard imports when number of imports from package > 99. And if that happens, we got bigger issues.

Comment (designatednerd) [reply android-studio-style-guide.md:13]: I don't think that's a default setting - i think the default is something like 3

Comment (csdodd) [reply android-studio-style-guide.md:13]: Added instructions for Java

**Review by designatednerd — APPROVED** [review]
Seems legit to me

**Review by anskaal — APPROVED** [review]

**Comment by csdodd** [comment]
I know Martin and Joseph are in favour of this as well - so majority rules. I'll merge.

**Pull request merged**